Formula/md5sha1sum.rb

```diff
@@ -1,7 +1,7 @@
 class Md5sha1sum < Formula
   desc "Hash utilities"
-  homepage "http://www.microbrew.org/tools/md5sha1sum/"
```

Comment:
this url still seems valid

Reply:
Hm, this one fails from my home internet connection since at least 6 month, but now I tried it from 4G and an external server and there it's okay. Not sure what may be the reason — correction committed.

Reply:
Looks as if my landline IP range was blocked.